tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1330,6 +1330,151 @@ def assert_shape_match(inp, out):
     return (res_outputs, new_state)
 
 
+class LayerNormBasicGRUCell(rnn_cell_impl._LayerRNNCell):
+  """GRU unit with layer normalization.
+
+    This class adds layer normalization to a
+    basic GRU unit. Layer normalization implementation is based on:
+
+      https://arxiv.org/abs/1607.06450.
+
+    "Layer Normalization"
+    Jimmy Lei Ba, Jamie Ryan Kiros, Geoffrey E. Hinton
+
+    and is applied before the internal nonlinearities.
+    """
+
+  def __init__(self, num_units,
+               activation=math_ops.tanh,
+               layer_norm=True,
```

Comment:
isn't it always true?

Reply:
i.e., if users don't want it, they can just use regular GRUCell, right?

Reply:
Yes they can. In the future both cells could be merged. Meanwhile, I can remove the option. I find it probably easier and more convenient to switch a boolean value to activate/deactivate layernorm rather than changing cell, but your code you rules, tell me what you prefer.

Reply:
@ebrevdo can you comment on this?

Reply:
@ebrevdo can you comment on this?

Reply:
Let's keep these separate and assume layer_norm=true always

Reply:
Ok I removed the 'layer_norm' parameter and always apply layer normalization